.github/CODEOWNERS

```diff
@@ -15,6 +15,9 @@
 # Extensions
 /sdk/extensions/                                @pakrym
 
+# Compute
+/sdk/compute/                                   @bilaakpan-ms @sandido @dkulkarni-ms @haagha @madewithsmiles @MS-syh2qs @grizzlytheodore
```

Comment:
Would you mind replacing line 18 with `# PRLabel: %Compute`  or adding the PR line following so that pull requests are automatically tagged?

Reply:
Sounds great! I added it, that will be very useful. Thank you